packages/abi/src/coder/utils/createFunctionSelector.ts

```diff
@@ -0,0 +1,9 @@
+import { bufferFromString } from '@fuel-ts/crypto';
+import { sha256 } from '@fuel-ts/hasher';
+import { bn } from '@fuel-ts/math';
+
+export const createFunctionSelector = (functionSignature: string): string => {
```

Comment:
```suggestion
/**
 * Creates a function selector from a function signature. This acts as a unique identifier for the function
 * that can be used to identify the function in the contract. Previously, this was used in contract calls adhering
 * to the v0 specification. This has since been superseded to be a StdString representation of a function name.
 * However this selector still exists as a unique identifier for the function.
 *
 * @param functionSignature - The function signature to create a selector from.
 * @returns The function selector.
 */
export const createFunctionSelector = (functionSignature: string): string => {
```